docker-compose.yml

```diff
@@ -478,8 +480,8 @@ services:
       - _APP_EXECUTOR_SECRET
       - _APP_EXECUTOR_HOST
       - _APP_USAGE_STATS
-      - DOCKERHUB_PULL_USERNAME
-      - DOCKERHUB_PULL_PASSWORD
+      - APP_DOCKER_HUB_PULL_USERNAME
+      - APP_DOCKER_HUB_PULL_PASSWORD
 
```

Comment:
Not directly related to this PR but these vars should be prefixed with `_`

Reply:
resolved